src/server/app.js

```diff
@@ -6,8 +8,20 @@ import http from 'http';
 import logger from 'morgan';
 import session from 'express-session';
 
+import type { $Application, Router } from 'express';
+import type { Server } from 'http';
+
+import Authentication from './auth/auth';
+
 class App {
-  constructor(config) {
+  port: number;
+
+  express: $Application<>;
```

Comment:
Why empty `<>`?

Reply:
Flow complains of 0-2 arguments without it, per the community-built Express types.